test/performance/shmem_perf_suite/bw_common.h

```diff
@@ -196,7 +196,9 @@ void static command_line_arg_check(int argc, char *argv[],
                     "[-k (kilobytes/second)] [-b (bytes/second)] \n"\
                     "[-v (validate data stream)] \n");
         }
+#ifndef VERSION_1.0
```

Comment:
Actually, nevermind that comment.  I see now that this is for OpenSHMEM 1.0 compatibility, which is even before we had SHMEM_MAJOR/MINOR_VERSION..  :(  Ok, this is fine..